pyta.py

```diff
@@ -10,13 +10,14 @@
 """
 import importlib.util
 import pylint.lint as lint
-from pylint.reporters import BaseReporter
+from reporters.color_reporter import ColorReporter
 from astroid import MANAGER
 
 import webbrowser
 import sys
 
 
+
```

Comment:
Delete this line. Don't submit whitespace changes in your pull requests. You should be checking for this before committing your work.